nasa/harmony
# HARMONY-1443: Add multiple queue processors for updating work items to keep up with load

## Jira Issue ID
HARMONY-1443

## Description
Adds multiple update queue processors to keep up when processing many updates

## Local Test Steps
1. check out this branch
2. deploy to sandbox
3. issue a big query (change url to match your deployment)
```
http://internal-harmony-jn-1410-frontend-760118495.us-west-2.elb.amazonaws.com/C1234208438-POCLOUD/ogc-api-coverages/1.0.0/collections/alt_echo_type/coverage/rangeset?skipPreview=true&ignoreErrors=true
```
5. open the sandbox console and go to the SQS page.  verify that the queues stay at/near 0 messages available with occasional messages in flight showing up
6. verify that your job is progressing
7. perhaps issue a sequential request as well and verify that it completes fairly quickly

## PR Acceptance Checklist
